docs/core/get-started.md

```diff
@@ -47,27 +47,27 @@ You can get started developing .NET Core applications by following these step-by
 - Watch a video on [how to install and use Visual Studio 2017 and .NET Core](https://channel9.msdn.com/Blogs/dotnet/Get-Started-NET-Core-Visual-Studio-2017/).
 - [Getting started with .NET Core using the command-line.](tutorials/cli-create-console-app.md)
 
-See the [.NET Core dependencies and requirements](install/dependencies.md?tabs=netcore30&pivots=os-windows) article for a list of the supported Windows versions.
+See the [.NET Core dependencies and requirements](./install/dependencies.md?pivots=os-windows) article for a list of the supported Windows versions.
```

Comment:
Why add "./"?

Reply:
For me, it's easier to see is a sub-folder with the . 

But I can undo that if the team doesn't like it.

Reply:
I feel we should leave it a personal matter. Both are valid ways of the same thing.